package/CHANGELOG

```diff
@@ -32,6 +32,8 @@ Changes
 
 Deprecations
 
+04/02/22 HenokB
+  * Added a new badge to link it to the benchmark
```

Comment:
I'd reiterate @RMeli's tip to read the top of the file, specifically that `entries are sorted newest-first`.

The point of the CHANGELOG is to summarise changes between versions. The ??/??/?? date in the most recent entry is set when the next release of MDAnalysis is released, specifically 2.2. Your change will go into that release, so you should add it under that section.

That being said, I don't think you actually need to describe the adding of the badge in the CHANGELOG. It's for users to read so they know what affects their code, and the badge has no effect on the code. That being said you deserve credit for the improvement to MDA, so I do suggest you add your Github username to the existing line of usernames for the next release.
`